File: OOPs/week2/area.cpp
        #include <iostream>
        #include <math.h>
        using namespace std;

        float area(float radius){
        return 3.14 * radius * radius;
        }

        float area(float length, float breath){
        return length * breath;
        }

        float area(float a, float b, float c){
        float s = (a+b+c)/3;
        
        return sqrt(s*(s-a)*(s-b)*(s-c));
        }

        int main(){
        float radius = 5;
        float len = 15, bre = 7.5;
        float a = 5, b = 8, c = 9; 
        
        cout << "Area of a circle with radius " << radius << " is " << area(radius) << endl;
        cout << "Area of a rectangle with length " << len << " and breadth "<< bre << " is " << area(len,bre) << endl;
        cout << "Area of a Triangle with sides " << a << " " << b << " " << c << " is " << area(a,b,c) << endl;


        return 0;
        }

File: JAVA/Week2/Lab3/area.java
import java.util.Scanner;


 


public class area{
   
    double area(double radius){
        return 3.14 * radius * radius;
}

double area(double length, double breadth){
        return length * breadth;
}

double area(double a, double b, double c){
    double s = (a+b+c)/2;
    return Math.pow((s*(s-a)*(s-b)*(s-c)),0.5);
}
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
       
        area a1 = new area();

        System.out.print("Enter the radius of the circle: ");
        System.out.println("The Area of the circle is "+String.valueOf(a1.area(input.nextDouble())));

        System.out.print("Enter the length and breath of the rectangle: ");
        System.out.println("The Area of the rectangle is "+String.valueOf(a1.area(input.nextDouble(),input.nextDouble())));

        System.out.print("Enter the 3 sides of the triangle:");
        System.out.println("The Area of the triangle is"+String.valueOf(a1.area(input.nextDouble(),input.nextDouble(),input.nextDouble())));
            
    }
}